### ISSUE TITLE
kill player if they fall out of bounds

### ISSUE THREAD

**Issue by roboparker** [diaboloshogunate/Psyforce#30]
some of the level designs allow the player to fall through the floor. if they do, kill them.

**Comment by sirslade** [comment]
Thoughts on how to do this:
Either create a 2d collider below the level with the killingObject property
Or
Do a check on update for player controllers where if `transform y < -(large-ish number) kill`

**Comment by saurabhs** [comment]
A 2D collider will be much better, so that the "out of bounds" location can be modifeid without any change in the code.

**Comment by diaboloshogunate** [comment]
Agreed and thank you for checking out our game. This was for a global game jam a while back. 

I am open to people making pull request and merging them but I personally don't plan on making any changes to the game at this time.
